pom.xml

```diff
@@ -96,6 +96,7 @@
         <java.version>1.8</java.version>
         <jcache.ri.version>0.5</jcache.ri.version>
         <jersey.version>2.25</jersey.version>
+        <jooq.version>3.11.11</jooq.version>
```

Comment:
Maybe think about using the most recent version of [jOOQ](https://www.jooq.org/notes) which is [3.12.3](https://www.jooq.org/notes#3.12.3) since end of October.